macosx-aarch64/Makefile

```diff
@@ -0,0 +1,17 @@
+CC = cc
+SRC = alpn.c clientcert.c options.c session.c ssl.c threads.c util.c
+OBJ = $(patsubst %.c, target/%.o, $(SRC))
+
+default: target/classes/macosx-aarch64/libwfssl.dylib
+
+clean:
+	rm -rf target
+
+target/classes/macosx-aarch64:
+	mkdir -p target/classes/macosx-aarch64
+
+target/%.o : ../libwfssl/src/%.c target/classes/macosx-aarch64
+	$(CC) $(CFLAGS) -mmacosx-version-min=11.0 -target arm64-apple-macos11.0 -Werror -Wall -Wmissing-prototypes -Wstrict-prototypes -Wmissing-declarations -Wpointer-arith -std=c89 -Wdeclaration-after-statement -Wformat -Wformat-security -Wunused -Wno-unknown-pragmas -c $< -o $@ -I../libwfssl/include -I${JAVA_HOME}/include -I${JAVA_HOME}/include/darwin
```

Comment:
Is `-mmacosx-version-min=11.0 -target arm64-apple-macos11.0` needed? (The existing Makefile for `macosx-x86_64` didn't have something like this.)

Reply:
There are no arm64 versions of macos pre Big Sur (which is macos version 11)

Reply:
Yes I think so - versions before that did not have an arm64 version as far as I know.
The other three things you requested: done, done and done.